docsearchConfig.json

```diff
@@ -2,27 +2,31 @@
   "index_name": "docodile",
   "sitemaps": ["https://docs.wandb.ai/sitemap.xml"],
   "start_urls": ["https://docs.wandb.ai"],
-  "stop_urls": ["https://docs.wandb.ai/ja", "https://docs.wandb.ai/ko"],
+  "stop_urls": [
+    "https://docs.wandb.ai/ja",
+    "https://docs.wandb.ai/ko",
+    ".*_print.*"
+  ],
   "sitemap_alternate_links": false,
   "selectors": {
     "lvl0": {
-      "selector": "(//ul[contains(@class,'menu__list')]//a[contains(@class, 'menu__link menu__link--sublist menu__link--active')]/text() | //nav[contains(@class, 'navbar')]//a[contains(@class, 'navbar__link--active')]/text())[last()]",
+      "selector": "(//ul[contains(@class,'navbar_nav')]//a[contains(@class, 'nav_link active')]/text() | //ul[contains(@class,'ul_0')]//a[contains(@class, 'td-sidebar-link active')]/text())",
```

Comment:
Were you able to confirm that the nav is no longer being indexed as content, with this change?

Reply:
Oh, I thought John mentioned something about the search results seeming to show up multiple times, one for the content and one for the navigation. We speculated that maybe this code was trying to exclude the top and left nav themselves from showing up as search results. Maybe I completely misunderstood.

Reply:
Ah. Yeah, that existed before too when we were on Docusaurus 😬